### ISSUE TITLE
v0.4.9 => "Failed test 'Saving sketches"

### ISSUE THREAD

**Issue by tseemann** [lskatz/mashtree#37]
```
t/02_lambda.t ............. 2/6
    #   Failed test 'MD5 of sample1.fastq.gz.msh'
    #   at t/02_lambda.t line 96.
    #          got: 'f52e4232f5c7c46e77a15a9ac7ee1bfd'
    #     expected: 'b737ed5e87f4851181c0f3027848ab4b'

    #   Failed test 'MD5 of sample4.fastq.gz.msh'
    #   at t/02_lambda.t line 96.
    #          got: 'd4e868824b2bae710aaad61da87d13ec'
    #     expected: '53d545265d37632dbfcb0d2556e8aba6'

    #   Failed test 'MD5 of sample2.fastq.gz.msh'
    #   at t/02_lambda.t line 96.
    #          got: '68df1823233fbea03e82778b3c3c4a2e'
    #     expected: '90e5a975176b4add3eb13891a1ee8368'

    #   Failed test 'MD5 of sample3.fastq.gz.msh'
    #   at t/02_lambda.t line 96.
    #          got: '5d9db49d34e6193cfca45db1005613ef'
    #     expected: '52fbe9f503f240065def9872cfd8e308'
    # Looks like you failed 4 tests of 4.

#   Failed test 'Saving sketches'
#   at t/02_lambda.t line 99.
# Looks like you failed 1 test of 6.
t/02_lambda.t ............. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/6 subtests
```

**Comment by lskatz** [comment]
I discovered that the sketch contains an absolute path to the original fastq file, making everyone's sketches unique.  Therefore this was not a good test in the script.  If this was your only error, then you can be worry-free.  I will work to resolve this issue over the next week when I have time.

**Comment by lskatz** [comment]
fixed in 5900f9a31e8da6d9ebe0f73a7ac327a69059fba8 but I don't have a new cpan release yet due to limited time
